.travis.yml

```diff
@@ -1,22 +1,23 @@
-language: java
-before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-deploy:
-  on:
-    branch: master
-  provider: elasticbeanstalk
-  access_key_id:
-    $AWS_ACCESS_KEY_ID
-  secret_access_key:
-    $AWS_SECRET_ACCESS_KEY
-  region: "eu-west-1"
-  app: "Gilded Rose Pairing Project"
-  env: "Gildedrosepairingproject-env"
-  bucket_name: "gilded-rose-pairing-project"
-  skip_cleanup: true
-  zip-file: build/libs/GildedRoseKata-0.0.1-SNAPSHOT.jar
+jobs:
+  include:
+    - name: "Backend"
+      language: java
+      before_script:
+        - cd backend
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+      cache:
+        directories:
+          - $HOME/.gradle/caches/
+          - $HOME/.gradle/wrapper/
+    - name: "Frontend"
+      language: java
```

Comment:
I would look into Travis's built in JS builders, if that's available.

Reply:
Yes I played around with the built in node.js language but kept getting errors.

Reply:
Hence the four failing commits above

Reply:
I finally figured out how to use the travis built in node.js language